배수현/세션_1/src/CounterPage.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+const CounterPage = ({counter, setCounter}) => {
+
+  const increment = () => {
+    setCounter(counter + 1)
+  }
+
+  const decrement = () => {
+    setCounter(counter - 1)
+  }
```

Comment:
setCounter를 받는 것보다 핸들러 함수도 상위에서 내려주는 건 어떠신가요? setCounter를 직접 넘겨주는 것보다 핸들러 함수를 넘겨줘서 사용하는 것이 안정성 면에서 더 좋다고 하더라구요!
```suggestion
const CounterPage = ({increase, decrease}) => {

```